src/internal/components/option/interfaces.ts

```diff
@@ -38,6 +38,7 @@ export interface DropdownOption {
   disabled?: boolean;
   disabledReason?: string;
   option: OptionDefinition | OptionGroup;
+  afterHeader?: boolean;
```

Comment:
Added this for the sticky option to be aware of whether there is content in the header slot of the list, because in this case it shouldn't have top border:

![Screenshot 2025-03-21 at 12 16 44](https://github.com/user-attachments/assets/c8fd6fce-a373-4894-9fbd-1ab24cf81631)

The motivation for this is the Multiselect filter in combination with the "Select all" feature:

![Screenshot 2025-03-21 at 12 20 33](https://github.com/user-attachments/assets/3697f378-b685-40c5-824f-b09393b44c27)